ui/k-info-panel/switch_info.kytos

```diff
@@ -55,13 +63,42 @@
        this.get_flows()
      },
      get_flows() {
+       this.nextState = this.metadata.enabled == 'true'? 'Disable': 'Enable'
```

Comment:
@Alopalao updating the next state on this `get_flows` function here is mixing the responsibility of this original function, if you meant to initialize a value maybe do it in the `mounted` function would be more appropriate, let me know if you had other original intention here. though

Reply:
Now `nex_state `value is initialize inside mounted. Because of problems with the virtual machine I had to corrected it. Commit [d118cd4740e4ef4606d1672b2628156cc359e960](https://github.com/kytos-ng/topology/pull/70/commits/d118cd4740e4ef4606d1672b2628156cc359e960)